### ISSUE TITLE
Magic skills gained from Light/Dark arts does not match what is displayed if it's not high enough.

### ISSUE THREAD

**Issue by almuth150** [DerpyProjectGroup/topaz#534]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened
- [x] I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I have specified what branch this happens on branch: base

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 
I've been testing with Drain due to how skill can be figured out by reversing the damage formula, since it uses solely skill for damage. With 75 RDM/SCH and Dark Arts active and 8 dark magic merits, the client says I have 262 dark magic skill with no skill from other sources, however the damage return from Drain is as if I had ~229 skill (rounding issues). However, if I equip my dark skill+ gear to get up to 300, the damage return is about as expected (maybe a point of skill short? Hard to tell because again, rounding issues). With WHM/SCH, it says I have 262 skill but returns as if I had 220 skill. Again, increasing skill via gear reduces the difference.

tl;dr mismatch between how much skill the client says I have and the skill used to determine damage from Drain.
